prow/config.yaml

```diff
@@ -18187,6 +18280,9 @@ periodics:
       ports:
       - containerPort: 9999
         hostPort: 9999
+      resources:
+        requests:
+          memory: "2Gi"
```

Comment:
Should this be in the `volumes` section? What about`ports`?
The changes below this are similar.

Reply:
no? the jobs are OOM because they run out of ram. disk is not an issue currently.

Reply:
https://github.com/krzyzacy/test-infra/blob/42f32bcdcd1a7a12d43fabf3afb1da84b0c4092d/prow/kube/types.go#L133 resources is defined within container

Reply:
Sorry, I wasn't clear. This PR is **currently** adding `resources` and `ports` under `volumes` in some places.

Reply:
grrr, I think the config is correct, but git UI is not expanding properly

Reply:
xref comment below about a new presubmit enforcing this: https://github.com/kubernetes/test-infra/pull/5475#discussion_r150658844